features/config/prompt.feature

```diff
@@ -1,26 +0,0 @@
-Feature: Automatically running the configuration wizard if Git Town is unconfigured
```

Comment:
Most of the Scenarios in this test are actually failing, so I extracted them into dedicated feature specs in the folders for the commands. The advantage of doing so is that now the tests can set up the environment so that the respective command works and verify that Git Town continues to execute the actual command logic after asking the user for the missing configuration info.

Reply:
Cool